### PR TITLE
Add err handling

### PR DESCRIPTION
**Pull request by yuxiaobo96**
Signed-off-by: yuxiaobo <yuxiaobogo@163.com>

### CODE REVIEW

File: tests/e2e/manifests/verify_manifest_lists.go [hunk 1]
```diff
@@ -230,7 +230,9 @@ func getFromURLTimeoutSize(url string, timeout int, sizeOnly bool) (string, int,
 	if err != nil {
 		return "", -1, err
 	}
-	defer resp.Body.Close()
+	defer func() {
+		_ = resp.Body.Close()
+	}()
```
Comment (neolit123) [inline tests/e2e/manifests/verify_manifest_lists.go]: why is this change needed?
does a linter complain about the lack of return value?

Comment (yuxiaobo96) [reply tests/e2e/manifests/verify_manifest_lists.go]: I just saw a document with similar error handling in it.
https://travix.io/unhandled-errors-in-go-3f341f2704dd
I am very sorry, maybe I didn't understand.

Comment (neolit123) [reply tests/e2e/manifests/verify_manifest_lists.go]: no problem, but given golint is not reporting this as error i don't think we should include the change.

even if `resp.Body.Close()` fails we don't need to handle the error it will return.
so `defer resp.Body.Close()` seems fine.

Comment (yuxiaobo96) [reply tests/e2e/manifests/verify_manifest_lists.go]: Ok, I understand it.
Thank you for your patience to explain this.
:)
